### ISSUE TITLE
`const` constructor support for fixed size strings

### ISSUE THREAD

**Issue by RivenSkaye** [chuckcscccl/fixedstr#7]
I'm currently facing a challenge with using the `tstr` and by extension the `strN` types in `const` contexts in a situation where I need to store default prefixes for further processing; Looking at the code in `tiny_internal.rs` it should be possible to refactor this to turn `make` into a `const fn`.

It _should_ be relatively simple with a loop similar to the one commented out already. Would a PR changing either `make` or `create`, or adding a `const` variant of those functions be accepted to allow for `const` contexts?

**Comment by chuckcscccl** [comment]
I've just published version 0.5.5, which has const constructors for several of the fixed string types, including tstr.  I added new constructors and the left the original ones in place because I can't call the copy_from_slice function, which is recommended by the Rust performance lints, inside a const function. Thus the original ones should still be better when called from a non-const context. In addition, I've also made the .len() and new() functions const, so the following is now possible:
```
const C:str16 = str16::const_make("abcd");
let A = [0u8;C.len()];
```

When I have more time I may make some of the other functions const.
